src/components/PostCommentEditor.vue

```diff
@@ -85,8 +85,11 @@ export default {
       return this.post.name_type !== 0 && this.post.is_mine ? 'author_red' : ''
     },
     isVerified () {
+      if (!this.post.my_comment_profile) {
+        return false
+      }
       const profile = this.post.my_comment_profile.profile
-      return this.post.parent_board.id === 14 ? profile?.is_school_admin : profile?.is_official
+      return this.post.parent_board?.id === 14 ? profile.is_school_admin : profile.is_official
```

Comment:
1. `parent_board.id`를 magic number인 14와 비교하는 것은 맞지 않습니다. 현재 백 코드가 이렇게 넘겨주는 게 문제인데 추후 수정하겠습니다.
2. `parent_board`가 없을 경우 조건이 거짓이 되어 `is_official`을 반환하게 됩니다.

(91번째 줄을 삭제하고 일부 수정했습니다.)

```suggestion
      const {is_school_admin, is_official} = this.post.my_comment_profile.profile
      return this.post.parent_board?.id === 14 ? is_school_admin : is_official
```